sphinx/util/osutil.py

```diff
@@ -241,17 +241,10 @@ def abspath(pathdir):
     return pathdir
 
 
-def getcwd():
```

Comment:
To keep compatibility, please don't remove this. Unfortunately some extensions might has used this.
To do that, please do following procedure:

1. Show deprecation message using `warnings.warn()` and `RemovedInSphinx40Warning` inside this function.
2. Move this to `sphinx.util.compat`.
3. Import it on this module.